gitbi/git_bisector.py

```
@@ -108,52 +113,50 @@ def run_git_bisect(
-                    commit_match = result.stdout.split('\n')[0].split(':')[0].strip()
-                    return commit_match
+                # Set up git bisect
+                subprocess.run(['git', 'bisect', 'start'], check=True)
+                subprocess.run(['git', 'bisect', 'good', start_commit], check=True)
+                subprocess.run(['git', 'bisect', 'bad', end_commit], check=True)
                 
-                if 'There are no more revisions' in result.stdout:
-                    raise ValueError("No change detected between commits")
+                for _ in range(MAX_ITERATIONS):
+                    # Compare outputs
+                    if self.are_outputs_identical(baseline_output, current_output):
```

Ruff (0.8.2)

Undefined name `current_output`

(F821)